index.html

```diff
@@ -0,0 +1,181 @@
+                <li class="year">2015</li>
+            </ul>
+            </div>
+            <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <button>HTML</button>
+                <button>CSS</button>
+                <button>javaScript</button>
+            </div>
+            <button class="see-proj">See project</button>
+        </div>
+
+        <div class="work-tonic">
+            <div>
+                <img src="images/Snapshoot2.png" alt="Snapshoot2">
+            </div>
+            <h1>Multi-Post
+                Stories</h1>
+            <div class="text-info">
+                <ul>
+                <li class="client-name">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            </div>
+            <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <button>HTML</button>
+                <button>CSS</button>
+                <button>javaScript</button>
+            </div>
+            <button class="see-proj">See project</button>
+        </div>
+        <div class="work-tonic">
+            <div>
+                <img src="images/Snapshoot3.png" alt="Snapshoot3">
+            </div>
+            <h1>Tonic</h1>
+            <div class="text-info">
+                <ul>
+                <li class="client-name">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            </div>
+            <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <button>HTML</button>
+                <button>CSS</button>
+                <button>javaScript</button>
+            </div>
+            <button class="see-proj">See project</button>
+        </div>
+        <div class="work-tonic">
+            <div>
+                <img src="images/Snapshoot4.png" alt="snapshoot4">
+            </div>
+            <h1>Multi-Post
+                Stories</h1>
+            <div class="text-info">
+                <ul>
+                <li class="client-name">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            </div>
+            <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <button>HTML</button>
+                <button>CSS</button>
+                <button>javaScript</button>
+            </div>
+            <button class="see-proj">See project</button>
+        </div>
+    </section>
+                                 <!-- start About me -->
+    <section class="about-me">
```

Comment:
- [ ] Please make sure to position the elements inside the about section using `display:flex`, as per requirement.